app/controllers/base_controller.rb

```diff
@@ -1,4 +1,3 @@
-require 'spree/core/controller_helpers/respond_with'
```

Comment:
I think this fix this issue https://github.com/openfoodfoundation/openfoodnetwork/issues/6023

This problem I think is that we're including twice `Spree::Core::ControllerHelpers::RespondWith`

Once in [include Spree::Core::ControllerHelper](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/spree/core/controller_helpers.rb#L10)

Reply:
I had a look at this last week, and did some debugging. When I removed the `require` it seemed like the methods from the Spree gem version of this file were being used instead of our modified version...

Reply:
Makes sense, I didn't really tested on real life. This just fixed the specs. There are so much work to do man ☺️